### PR TITLE
FoxitReader: Fix checkver and url

### PR DESCRIPTION
**Pull request by linsui**
https://github.com/lukesampson/scoop-extras/issues/2308

### CODE REVIEW

**Comment by Ash258** [comment]
Why did you changed URL? Installation works fine. I installed it yesterday without problems.

File: bucket/foxit-reader.json [hunk 1]
```diff
@@ -1,29 +1,33 @@
...
+    "hash": "432dcea5ac97ac9198b042d7ec463dcf9cfd28541e5ebc4ce4672f4163498162",
     "bin": "FoxitReader.exe",
     "shortcuts": [
         [
             "FoxitReader.exe",
             "Foxit Reader"
         ]
     ],
+    "installer": {
+        "script": [
```
Comment (Ash258) [inline bucket/foxit-reader.json]:
```suggestion
        "script": [
            "Get-ChildItem \"$dir\" '*_Prom.exe' | Select-Object -ExpandProperty Fullname | Expand-InnoArchive -DestinationPath \"$dir\"",
            "Remove-Item \"$dir\\`$PLUGINSDIR\" -Recurse"
        ]
```

**Comment by linsui** [comment]
But the checkver is updated. Soon or later we need to change the url unless we can find some way to match the md5 in the url. Maybe we can catch the update request?

File: bucket/foxit-reader.json [hunk 2]
```diff
@@ -1,29 +1,33 @@
...
+            "$majorVersion = $version.Split('.') | Select-Object -first 1",
+            "$minorVersion = $version.Split('.') | Select-Object -skip 1 -first 1",
+            "Expand-InnoArchive \"$dir\\`$PLUGINSDIR\\FoxitReader${majorVersion}${minorVersion}_enu_Setup_Prom.exe\" \"$dir\"",
+            "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse"
+        ]
+    },
     "checkver": {
         "url": "https://www.foxitsoftware.com/downloads/downloadForm.php?retJson=1&product=Foxit-Reader&platform=Windows&language=English",
-        "regex": "\"version\":\\[\"([\\d.]+)\".+win\\\\\\/[\\d.]+\\\\\\/(?<hash>[A-F\\d]{32})"
+        "jp": "$.package_info.version[0]"
```
Comment (Ash258) [inline bucket/foxit-reader.json]:
```suggestion
        "jsonpath": "$.package_info.version[0]"
```

**Pull request merged**